app/views/layouts/application.html.erb

```diff
@@ -18,6 +18,14 @@ http://digitalnz.org/supplejack
     <%= javascript_include_tag "application" %>
     <%= csrf_meta_tags %>
     <%= yield(:head) %>
+
+    <%= favicon_link_tag asset_path('favicon-32x32.png'), rel: 'icon', type: 'image/png' %>
+    <%= favicon_link_tag asset_path('favicon-16x16.png'), rel: 'icon', type: 'image/png' %>
+    <%= favicon_link_tag asset_path('apple-touch-icon.png'), rel: 'apple-touch-icon', type: 'image/png', sizes: '144x144' %>
+    <%= favicon_link_tag asset_path('manifest.json'), rel: 'manifest' %>
+    <%= favicon_link_tag asset_path('safari-pinned-tab.svg'), rel: 'mask-icon', color: "#118a0f" %>
```

Comment:
is there a way around using the color hex? `color: "#118a0f"`

Reply:
Don't think so.